tfhe/src/core_crypto/algorithms/lwe_bootstrap_key_conversion.rs

```diff
@@ -96,6 +110,62 @@ pub fn convert_standard_lwe_bootstrap_key_to_fourier_mem_optimized<Scalar, Input
         .fill_with_forward_fourier(input_bsk.as_view(), fft, stack);
 }
 
+pub fn par_convert_standard_lwe_bootstrap_key_to_fourier_mem_optimized<
+    Scalar,
+    InputCont,
+    OutputCont,
+>(
+    input_bsk: &LweBootstrapKey<InputCont>,
+    output_bsk: &mut FourierLweBootstrapKey<OutputCont>,
+    fft: FftView<'_>,
```

Comment:
this function is not required as it's not taking buffers, no need for a mem optimized version here

Reply:
Removed